### PR TITLE
Correctly render tab stops in `--show-all` 

### PR DESCRIPTION
**Pull request by Synthetica9**
New behaviour:

```
───────┬────────────────────────────────────────────────────────────
       │ STDIN
───────┼────────────────────────────────────────────────────────────
   1   │ ··├┤␊
   2   │ ·├─┤␊
   3   │ ├──┤␊
   4   │ 12345678␊
───────┴────────────────────────────────────────────────────────────
```

Old behaviour

```
───────┬────────────────────────────────────────────────────────────
       │ STDIN
───────┼────────────────────────────────────────────────────────────
   1   │ ··├──┤␊
   2   │ ·├──┤␊
   3   │ ├──┤␊
   4   │ 12345678␊
───────┴────────────────────────────────────────────────────────────
```

Does not work with multi-width characters, yet, is this a requirement? If so, how should this work with unicode escaped characters?

### CODE REVIEW

**Comment by Enselic** [comment]
Nice 👍 I skimmed the code and it looks reasonable. Before a detailed review, can you please add at least one test (preferably more as long as it makes sense) for this in https://github.com/sharkdp/bat/blob/master/tests/integration_tests.rs? I.e. invoke the binary and assert on stdout.

That way we can be sure that this contribution and use case will never stop working.

**Comment by Synthetica9** [comment]
Okay, added two integration tests. Not sure about the names, feel free to suggest better ones.

**Comment by sharkdp** [comment]
Very nice! I like it.  

Could you please add an entry to the CHANGELOG? See instructions here: https://github.com/sharkdp/bat/blob/master/CONTRIBUTING.md#add-an-entry-to-the-changelog

File: tests/integration_tests.rs [hunk 1]
```diff
@@ -1153,6 +1153,52 @@ fn show_all_mode() {
...
+55555├─┤?␊
+666666├┤?␊
+7777777↹?␊
+88888888├──┤?␊
+",
+        );
+}
+
+#[test]
+fn show_all_tabstops_long() {
```
Comment (sharkdp) [inline tests/integration_tests.rs:1180]: Feel free to choose very long test names (I have to admit that some other test names could be improved as well). Maybe:

`show_all_extends_tab_markers_to_next_tabstop`

**Pull request merged**